index.html

```diff
@@ -132,15 +132,37 @@ <h3 class="worktitle">My Recent Work</h3>
                     <li><img src="./pictures/Rectangle70.png" alt=""></li>
                     <li>Ruby</li>
                 </ul>
-                    <button type="button" onclick="location.href='https://www.google.com/'">See Project</button>
+                    <button type="submit" onclick="openpopUp1('project1')" class="btn1">See Project</button>
+                </div>
+                <div class="popUp1 popUpactive" id="popUp1">
+                    <div class="marginpopUp">
+                    <div class="toppopup1">
+                        <p class="work1title" id="popuptitle"></p> 
+                        <button type="submit" class="btn2">&times; </button>
+                    </div>
+                    <img src="#" class="img_popUp1" id="popupimg" alt="">
+                    <p class="workdescription" id="popupdescription"></p>
+                    <ul class="usedcode">
+                        <li class="pop" id="popuptec1"></li>
+                        <li class="pop" id="popuptec2"></li>
+                        <li class="pop" id="popuptec3"></li>
+                        <li class="pop" id="popuptec4"></li>
+                    </ul>
+                    <div class="popup_buttons">
+                    <a id="see_live1" href="" >
+                        <p>See live</p><img src="pictures/TimeAndIcons/github.png" alt="Github icon and link to account"></a>
+                    <a id="see_source1" href="">
+                        <p>Source</p><img src="pictures/TimeAndIcons/github.png" alt="Github icon and link to account"></a>
```

Comment:
- [ ] As the previous reviewer has suggested, kindly remove all the hard-coded data you have in your index.html file for the model as well since you are already generating it in the js file. 👍. The goal is to render the projects dynamically so all hard-coded data should be removed.